go/stats/export.go

```diff
@@ -229,6 +231,53 @@ func (v *Int) String() string {
 	return strconv.FormatInt(v.i.Get(), 10)
 }
 
+// Help returns the help string
+func (v *Int) Help() string {
```

Comment:
For all these simple types `Int` is same as `IntGauge`. We might as well have only one of them. Maybe we can just rename `Int` to `IntGauge`.